### ISSUE TITLE
FIX: Windows installation script need CUDA DLL version update

### ISSUE THREAD

**Issue by E3V3A** [alievk/avatarify#129]
The reason you get the error in #70, is that the windows installation script is using the wrong CUDA version. Since the latest version is 10.2.8

* IF you are going to use the *torch* version of CUDA, then leave as is, but
* IF you are going to use the latest CUDA from updated *torch* and *torchvision* or windows nVidia installation, then you need to change the line as show below.

[This line](https://github.com/alievk/avatarify/blob/master/scripts/install_windows.bat#L6) need to use `cuda102` instead of `cuda100` and read: 
```powershell
# either like this
call conda install -y pytorch==1.5.0 torchvision==0.6.0 cuda102 -c pytorch
# or like this:
call conda install -y pytorch torchvision cudatoolkit=10.2 -c pytorch
```
(See [change](https://github.com/alievk/avatarify/commit/1c502b97745a1320176e143f3fa2f9fb48c2b03b#diff-7cfecbd43999ac0b06993fc2ff3cb12b).)

Either way, this also seem to work with Python3.8.
I also installed: *`cupy-cuda102, cudatoolkit, numba`*. 

@alievk 
Remind me again, why we're not using updated torch and py3.8? (it works fine for me.)

**Comment by alievk** [comment]
We decided to stick to Pytorch 1.0.0 as per FOMM requirements for a time being. This version is not available for Python 3.8. You can easily check that by creating Python 3.8 conda env and trying to install Pytorch 1.0.0.

**Comment by E3V3A** [comment]
Hi @alievk 
I still don't see the issue. What do you mean with `FOMM requirements`?
The only thing I can find is what you [write here](https://github.com/alievk/first-order-model#training):

> Note: It is important to use pytorch==1.0.0 for training. Higher versions of pytorch have strange bilinear warping behavior, because of it model diverge.

* So it's only an issue for training? 
* Now I find it hard to believe that the latest torch hasn't fixed that bug in the 1.5 year old torch version (1.0.0 @ Dec 2018).  Have you tested this lately?
**Can you refer/link to the issue in that repo, for this bug?**


I am using miniconda3 (p3.7), with avatarify environment where I use Py3.8. 
Almost all my packages are up to date and it just works. 

Here are some details:
```bash
(avatarify)  $ python.exe .\collect_env.py
Collecting environment information...
-------------------------------------------------------------------------------
OS info:
    OS                  :  Microsoft Windows 8.1
    GCC/CL version      :  19.16.27026.1
    CMake version       :  3.17.0
    NVCC version        :  10.2.89
    MSVS version        :  15.9.17+28307.905

nVidia info:
    Graphics Card       :  GeForce GTX 850M
    Driver version      :  442.19
    CUDA availability   :  Yes
    CUDA version        :  10.2
    cuDNN version       :  10.2.86
    GPU configuration   :  GPU 0: GeForce GTX 850M

Conda info:
    Name                :  miniconda3
    Conda Version       :  conda 4.8.3
    Conda Path          :  C:\Users\xxxx\miniconda3\Scripts\conda.exe
    Conda Python Path   :  C:\Users\xxxx\miniconda3\python.exe
    Active Environment  :  avatarify

PyTorch info:
    Build version       :  1.5.0
    Debug build         :  No
    CUDA build          :  10.2

Python info:
    Environment         :  --
    Env.version         :  3.8
    PIP version         :  pip3

Relevant pip/conda packages & library versions:
     [pip3] numpy==1.18.2
    [conda] blas                      1.0                         mkl
    [conda] cudatoolkit               10.2.89              h74a9793_0
    [conda] mkl                       2020.0                      166
    [conda] mkl-service               2.3.0            py38hb782905_0
    [conda] mkl_fft                   1.0.15           py38h14836fe_0
    [conda] mkl_random                1.1.0            py38hf9181ef_0
    [conda] numpy                     1.18.1           py38h93ca92e_0
    [conda] numpy-base                1.18.1           py38hc3f5095_1
    [conda] pytorch                   1.5.0           py3.8_cuda102_cudnn7_0    pytorch
    [conda] torchvision               0.6.0                py38_cu102    pytorch
-------------------------------------------------------------------------------
```

and 

```bash
(avatarify)  $ conda list
# packages in environment at C:\Users\xxxx\miniconda3\envs\avatarify:
#
# Name                    Version                   Build  Channel
blas                      1.0                         mkl
ca-certificates           2020.1.1                      0
certifi                   2020.4.5.1               pypi_0    pypi
chardet                   3.0.4                    pypi_0    pypi
cloudpickle               1.3.0                      py_0
cryptography              2.9.2                    pypi_0    pypi
cudatoolkit               10.2.89              h74a9793_0
cupy-cuda102              7.4.0                    pypi_0    pypi
cycler                    0.10.0                   py38_0
cytoolz                   0.10.1           py38he774522_0
dask-core                 2.14.0                     py_0
decorator                 4.4.2                      py_0
face_alignment            1.0.1                      py_1    1adrianb
fastrlock                 0.4                      pypi_0    pypi
freetype                  2.9.1                ha9979f8_1
hdf5                      1.10.4               h7ebc959_0
icc_rt                    2019.0.0             h0cc432a_1
icu                       58.2                 ha66f8fd_1
idna                      2.9                      pypi_0    pypi
imageio                   2.8.0                      py_0
intel-openmp              2020.0                      166
joblib                    0.14.1                   pypi_0    pypi
jpeg                      9b                   hb83a4c4_2
kiwisolver                1.0.1            py38ha925a31_0
libopencv                 4.0.1                hbb9e17c_0
libpng                    1.6.37               h2a8f88b_0
libtiff                   4.1.0                h56a325e_0
llvmlite                  0.32.0           py38ha925a31_0
matplotlib                3.2.1                    pypi_0    pypi
matplotlib-base           3.1.3            py38h64f37c6_0
mkl                       2020.0                      166
mkl-service               2.3.0            py38hb782905_0
mkl_fft                   1.0.15           py38h14836fe_0
mkl_random                1.1.0            py38hf9181ef_0
networkx                  2.4                        py_0
ninja                     1.9.0            py38h74a9793_0
numba                     0.49.0           py38h47e9c7a_0
numpy                     1.18.1           py38h93ca92e_0
numpy-base                1.18.1           py38hc3f5095_1
olefile                   0.46                       py_0
opencv                    4.0.1            py38h2a7c758_0
opencv-python             4.2.0.34                 pypi_0    pypi
openssl                   1.1.1g               he774522_0
pandas                    1.0.3                    pypi_0    pypi
pillow                    7.0.0            py38hcc1f983_0
pip                       20.0.2                   py38_1
py-opencv                 4.0.1            py38he44ac1e_0
pygit                     0.1                      pypi_0    pypi
pyparsing                 2.4.7                    pypi_0    pypi
pyqt                      5.9.2            py38ha925a31_4
python                    3.8.2               h5fd99cc_11
python-dateutil           2.8.1                      py_0
pytorch                   1.5.0           py3.8_cuda102_cudnn7_0    pytorch
pytz                      2019.3                   pypi_0    pypi
pywavelets                1.1.1            py38he774522_0
qt                        5.9.7            vc14h73c81de_0
requests                  2.23.0                   pypi_0    pypi
scikit-image              0.16.2           py38h47e9c7a_0
scikit-learn              0.22.2.post1             pypi_0    pypi
scipy                     1.4.1            py38h9439919_0
setuptools                46.1.3                   py38_0
sip                       4.19.13          py38ha925a31_0
six                       1.14.0                   py38_0
sqlite                    3.31.1               he774522_0
tbb                       2020.0               h74a9793_0
tk                        8.6.8                hfa6e2cd_0
toolz                     0.10.0                     py_0
torchvision               0.6.0                py38_cu102    pytorch
tornado                   6.0.4            py38he774522_1
tqdm                      4.45.0                     py_0
urllib3                   1.25.9                   pypi_0    pypi
vc                        14.1                 h0510ff6_4
virtualenv                20.0.18                  pypi_0    pypi
vs2015_runtime            14.16.27012          hf0eaf9b_1
wheel                     0.34.2                   py38_0
wincertstore              0.2                      py38_0
xz                        5.2.5                h62dcd97_0
zlib                      1.2.11               h62dcd97_4
zstd                      1.3.7                h508b16e_0
```

and pip (pip3.8 in conda)

```bash
(avatarify)  $ pip list
Package          Version
---------------- -------------------
appdirs          1.4.3
certifi          2020.4.5.1
cffi             1.14.0
chardet          3.0.4
cloudpickle      1.3.0
cryptography     2.9.2
cupy-cuda102     7.4.0
cycler           0.10.0
cytoolz          0.10.1
dask             2.14.0
decorator        4.4.2
distlib          0.3.0
face-alignment   1.0.1
fastrlock        0.4
filelock         3.0.12
idna             2.9
imageio          2.8.0
Jinja2           2.11.1
joblib           0.14.1
kiwisolver       1.0.1
llvmlite         0.32.0
MarkupSafe       1.1.1
matplotlib       3.2.1
mkl-fft          1.0.15
mkl-random       1.1.0
mkl-service      2.3.0
networkx         2.4
numba            0.49.0
numpy            1.18.1
olefile          0.46
opencv-python    4.2.0.34
openssl-python   0.1.1
pandas           1.0.3
Pillow           7.0.0
pip              20.0.2
pipenv           2018.11.26
pycparser        2.20
pygit            0.1
pyparsing        2.4.7
python-dateutil  2.8.1
pytz             2019.3
PyWavelets       1.1.1
PyYAML           5.3.1
requests         2.23.0
scikit-image     0.16.2
scikit-learn     0.22.2.post1
scipy            1.4.1
setuptools       46.1.3.post20200330
sip              4.19.13
six              1.14.0
toolz            0.10.0
torch            1.5.0
torchvision      0.6.0
tornado          6.0.4
tqdm             4.45.0
urllib3          1.25.9
virtualenv       20.0.18
virtualenv-clone 0.5.4
wheel            0.34.2
wincertstore     0.2
```

**Comment by eabase** [comment]
**FOMM** =*`First Order Motion Model`*

* https://papers.nips.cc/paper/2019/hash/31c0b36aef265d9221af80872ceb62f9-Abstract.html
